### ISSUE TITLE
add option to enable running natively under wayland

### ISSUE THREAD

**Issue by Randdose** [MrAlex94/Waterfox#2961]
**Is your feature request related to a problem? Please describe.**

> _A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]_

currently you have to export a global variable "MOZ_ENABLE_WAYLAND=1" to make waterfox run natively under wayland.

**Describe the solution you'd like**

> _A clear and concise description of what you want to happen._

it would be a lot more convenient if there was a toggle in the settings, that would export the variable and restart the browser.

**Describe alternatives you've considered**

> _A clear and concise description of any alternative solutions or features you've considered._

adding clear instructions on how to do this in an accessible place; waterfox.net/support/ website under linux section.

**Additional context**

> _Add any other context or screenshots about the feature request here._

**Comment by hawkeye116477** [comment]
You can just create shortcuts for running Waterfox at Wayland.

**Comment by Randdose** [comment]
@hawkeye116477 yes i already changed the .desktop file to run under wayland, but it would be more convenient with a toggle/button.

surprisingly chromium/chrome already does this; it has a drop down list under chrome://flags for choosing a display server.

I should probably request this feature from firefox, if this seems like a more appropriate way i can close this issue.
